src/v2/Components/FollowButton/FollowArtistButton.tsx

```diff
@@ -233,3 +236,32 @@ export const FollowArtistButtonFragmentContainer = createFragmentContainer(
     `,
   }
 )
+
+export const FollowArtistButtonQueryRenderer: React.FC<
```

Comment:
So this is because the featured artists aren't actual `Artists` they are `FeaturedLinks` — so I'm refetching them as actual artist follows using the parsed out slug... but your comment points out a minor bug where I'm passing the QueryRenderers the internalID, which is the slug most of the time but not all of the time. Will have to just match it out of the href instead.